py/fiberassign/test/test_qa.py

```diff
@@ -258,16 +258,30 @@ def test_science(self):
 
         self.assertGreaterEqual(frac,  99.0)
 
-        #- Test if qa-fiberassign script runs without crashing
-        bindir = os.path.join(os.path.dirname(fiberassign.__file__), '..', '..', 'bin')
-        script = os.path.join(os.path.abspath(bindir), 'qa-fiberassign')
-        if os.path.exists(script):
-            fafiles = glob.glob(f"{test_dir}/fiberassign-*.fits")
-            cmd = "{} --targets {}".format(script, " ".join(fafiles))
-            err = subprocess.call(cmd.split())
-            self.assertEqual(err, 0, f"FAILED ({err}): {cmd}")
-        else:
-            print(f"ERROR: didn't find {script}")
+        # FIXME:  These lines of code introduced fstring notation, which essentially
+        # requires python>=3.6 and is not used anywhere else in the code.  We could use
+        # fstring notation if we increase the python version requirement in setup.py.
+        # This mechanism for finding the qa-fiberassign script only works when running
+        # tests in place, and not when running installed tests with:
+        #
+        #  $> python -c 'import fiberassign.test; fiberassign.test.runtests()'
+        #
+        # The solution is to move qa-fiberassign into the fiberassign.scripts package
+        # so that it can be tested along with the other scripts (qa-fiberassign is
+        # a legacy script from before we refactored the package.)  Once that is done,
+        # Re-enable this test and call the entry point directly.
```

Comment:
f-string notation is fine; we now use that extensively in desispec and py3.6 is the minimum supported version now.

Auto-discovery of where `qa-fiberassign` is and actually running that is still desirable but doesn't have to be a blocking factor for this PR.  See [`desispec.test.test_binscripts`](https://github.com/desihub/desispec/blob/master/py/desispec/test/test_binscripts.py) for a multi-step discovery of where the scripts might be depending upon whether you are testing a bare git clone or an installed package.  And it has been occasionally useful to test that the actual script works, and not just the `desispec.scripts.blah` version of it works.

Reply:
Ok, I will bump the requirements to python >= 3.6.  Will look at the desispec code referenced and if it is quick to get in will do it before merge.